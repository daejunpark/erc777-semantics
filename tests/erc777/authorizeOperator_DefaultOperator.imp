test.preamble();

// Testing authorizeOperator(Operator) when Operator is a default operator.
// A token holder MAY authorize an already authorized operator. An AuthorizedOperator MUST be emitted each time.

test.authorizeOperator(8);
