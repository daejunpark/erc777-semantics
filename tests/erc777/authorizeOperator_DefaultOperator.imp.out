7 is authorizing 8 as an operator
